### PR TITLE
This is the branch containing the stuff from Friday's demo

### PR DESCRIPTION
**Pull request by hanseljt77**
There have been a few commits since then to clean things up. However, the primary focus of this branch is to add assigned case, positive pool, and log in functionality. Log in is delayed by backend stuff but the code is there. The interceptor in the app.module is  currently commented so it is effectively disabled until that's working.

### CODE REVIEW

File: src/app/app.component.html [hunk 1]
```diff
@@ -6,10 +6,28 @@
...
-    <mat-grid-tile routerLink="/dashboard">Dashboard</mat-grid-tile>
-    <mat-grid-tile routerLink="/userlist">User List</mat-grid-tile>
-    <mat-grid-tile routerLink="/positive-case">Positive Cases</mat-grid-tile>
-    <mat-grid-tile routerLink="/AssignedCases">Assigned Cases</mat-grid-tile>
+    <div [hidden]="!login.isAuthenticated()">
+      <mat-grid-tile routerLink="/dashboard">Dashboard</mat-grid-tile>
+    </div>
+    
+    <div [hidden]="!login.isAuthenticated() || login.roleIsPerson() || login.roleIsUnspecified()">
+      <mat-grid-tile routerLink="/userlist">User List</mat-grid-tile>
```
Comment (MCoriale) [inline src/app/app.component.html]: You mentioned that you wanted to change the column name from User List in the frontend, you would just need to change it here

**Review by MCoriale — DISMISSED** [review]
Minor changes-- 

Changing column names in the frontend 

- [x] User --> Person 
- [x] Did the 2x First Name columns get changed?

File: src/app/app.component.html (hunk 1 above)
Comment (hanseljt77) [reply src/app/app.component.html]: Done!

**Review by noahthomas88 — APPROVED** [review]
Jackson added these changes

**Pull request merged**